components/modal/Modal.tsx

```diff
@@ -144,6 +144,13 @@ const Modal: React.FC<ModalProps> = (props) => {
             maskTransitionName={getTransitionName(rootPrefixCls, 'fade', props.maskTransitionName)}
             className={classNames(hashId, className, modal?.className)}
             style={{ ...modal?.style, ...style }}
+            bodyStyle={{
```

Comment:
Personally, I think this is not a good solution.

Reply:
Thank your review @yoyo837. I am just starting to contribute. Can you please suggest me a nice fix so that i can go through and implement my ideas and give a better solution.